sbi/simulators/simutils.py

```diff
@@ -266,6 +284,20 @@ def check_prior_methods(prior):
             with prior.log_prob(theta). Consider using a PyTorch distribution."""
         )
 
+    assert hasattr(prior, "mean"), "the prior must have a mean attribute or property."
+    assert hasattr(
+        prior, "variance"
+    ), "the prior must have a variance attribute or property."
+
```

Comment:
As far as I know, the prior mean and variance are only used for standardizing `theta`. We could also make the standardization based on prior samples (and empirically estimate the mean and variance). Then we would not need these attribute checks anymore.

Or are mean and variance used somewhere else also?

Reply:
Yes, that would be a good solution!

Reply:
also a separate PR, maybe together with the new density estimator interface?

Reply:
Yes, sounds good

Reply:
this is now here #136